lib/galaxy_test/selenium/test_collection_edit.py

```diff
@@ -0,0 +1,60 @@
+from selenium.webdriver.common.keys import Keys
+
+from .framework import (
+    selenium_test,
+    SeleniumTestCase
+)
+
+
+class CollectionEditTestCase(SeleniumTestCase):
+
+    ensure_registered = True
+
+    @selenium_test
+    def test_change_dbkey_simple_list(self):
+        self.create_simple_list_collection()
+        self.switch_to_beta_history()
+        self.open_collection_edit_view()
+        self.navigate_to_database_tab()
+        dbkeyValue = "Additional"
+        self.check_current_dbkey_value(dbkeyValue)
+        dbkeyNew = "hg17"
+        self.change_dbkey_value_and_click_submit(dbkeyValue, dbkeyNew)
+        self.components.history_panel.options_use_legacy_history.wait_for_and_click()
+        self.history_panel_wait_for_hid_ok(4)
+        self.switch_to_beta_history()
+        self.open_collection_edit_view()
+        self.navigate_to_database_tab()
+        self.check_current_dbkey_value(dbkeyNew)
+
+    def create_simple_list_collection(self):
+        self.perform_upload(self.get_filename("1.fasta"))
+        self.history_panel_wait_for_hid_ok(1)
+
+        self.history_panel_multi_operations_show()
+        self.history_panel_muli_operation_select_hid(1)
+        self.history_panel_multi_operation_action_click(self.navigation.history_panel.multi_operations.labels.build_list)
+
+        self.collection_builder_set_name("my cool list")
+        self.screenshot("collection_builder_list")
+        self.collection_builder_create()
+        self.history_panel_wait_for_hid_ok(2)
+
+    def switch_to_beta_history(self):
```

Comment:
Please drop this and use ``self.use_beta_history()`` which is now available - if it isn't available I think you need a merge with the latest dev.